src/app/admin-page/admin-page.component.css

```diff
@@ -0,0 +1,13 @@
+#admin-actions-label {
+  margin: 0 0 25px 0;
+}
+
+.form-group {
```

Comment:
If you log out when on the admin page, the styles on the create account/login page are incorrect. For some reason, the overridden styles for `.form-group` and `.btn` still hold true after navigating via the router. One easy way to not have this happen is to use additional classes for the elements such as `.form-group-admin-page` and `.btn-admin-page`.